### ISSUE TITLE
Cannot reproduce the inpainting result

### ISSUE THREAD

**Issue by siujeong** [Xinjie-Q/Boosting-NeRV#1]
I cannot reproduce the inpainting results. I'm trying to reproduce the inpainting result with a disperse mask on the hnerv model using Boosting-NeRV/scripts/inpainting/hnerv.sh (on blackswan video from DAVIS2016)
I haven't modified any settings or code except for the data path. 
I want to reproduce the results shown below (results in the paper).

<img width="782" alt="image" src="https://github.com/Xinjie-Q/Boosting-NeRV/assets/133577494/c534fa65-f52a-4078-a0e8-49e841d3124f">

And this is my result. 

<img width="782" alt="image" src="https://github.com/Xinjie-Q/Boosting-NeRV/assets/133577494/89d78ce3-ba1c-4e75-a18f-f60d84501d67">

Can you help me with this, please?

**Comment by Xinjie-Q** [comment]
The hnerv model is unstable when training under inpainting. I apologize for not specifying the learning rate settings for different videos of hnerv under inpainting. During the inpainting experiment of hnerv, most videos used a learning rate of 0.001 (consistent with the original paper: https://github.com/haochen-rye/HNeRV), but some videos got NAN PSNR when lr=0.001. In this case, I would reduce the initial learning rate (7.5e-4, 5e-4, 2.5e-4, 1e-4). I will re-check the hyperparameter settings under scripts. 

The figure below shows the inpainting performance of the hnerv model on blackswan with a learning rate of 0.001 under NVIDIA RTX A6000.

![0aab8a815bd1d5c405be603f33d3553](https://github.com/Xinjie-Q/Boosting-NeRV/assets/64238111/fe561fd7-5650-43c4-94f5-7f2afc1cacef)
